frontend/src/component/operator-wizard/page-component/Page.js

```diff
@@ -128,8 +133,19 @@ export default class Page extends Component {
   };
 
   deploy = () => {
-    //alert("deploy here");
-    this.createSampleYaml();
+    var sampleYaml = this.createSampleYaml();
+    var result = this.createResultYaml(sampleYaml);
+    console.log(result);
+    fetch("/", {
+      method: "POST",
+      body: JSON.stringify(result),
+      headers: {
+        "Content-Type": "application/json"
```

Comment:
Are we posting json or yaml?

Reply:
i didnot look into deploy part.
As per Karel's comment, just moved this code from editYaml to Deploy functionality.

Reply:
result -consists of Yaml  thouh.

Reply:
ok, so look at how this file may have changed as a result of my previous PR, and reapply your changes accordingly.